hazelcast/src/main/java/com/hazelcast/spi/impl/proxyservice/impl/ProxyServiceImpl.java

```diff
@@ -173,7 +173,12 @@ public void destroyDistributedObject(String serviceName, String name, UUID sourc
             calls.add(f);
         }
 
-        destroyLocalDistributedObject(serviceName, name, source, true);
+        try {
+            destroyLocalDistributedObject(serviceName, name, source, true);
+        } catch (Throwable e) {
+            logger.severe("Destroying local DistributedObject failed", e);
```

Comment:
we reserve `severe` level for unrecoverable failures, I think `warning` is sufficient here.